sklearn/neighbors/_binary_tree.pxi

```diff
@@ -1164,7 +1164,7 @@ cdef class BinaryTree:
 
     def get_tree_stats(self):
         """
-        get_tree_stats(self)
+        get_tree_stats()
```

Comment:
Should this be the other way around where `query(...)` includes `self`?

(Our pure Python classes includes `self`)